src/components/Common/DateTimeInput.tsx

```diff
@@ -19,16 +19,18 @@ function toLocalDateTimeString(isoString: string | undefined): string {
 
 type DateTimeInputProps = React.ComponentProps<typeof Input> & {
   onDateChange?: (val: string) => void;
+  maxDate?: string;
```

Comment:
why not just re-use the in-built `max` prop that we get from props of Input component? 🤔